instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -50,6 +50,17 @@ def hello():
     application = get_wsgi_application()
     application = OpenTelemetryMiddleware(application)
 
+
+Usage in Production Server (gunicorn)
+------------------------------------
+
+Modify the ``gunicorn.config.py`` file as shown below.
+
+.. code-block:: python
+
+    def post_fork(server, worker):
```

Comment:
I have seen some problems regarding the forking process of Gunicorn when working with Opencensus like [here](https://github.com/census-instrumentation/opencensus-python/issues/957) and [here](https://github.com/census-instrumentation/opencensus-python/issues/950). The main issue was that the worker thread that was responsible for sending spans to the exporter (BatchSpanProcessor in this case) was not copied to the child processes due to Gunicorn using `os.fork()` to spawn them. Does this `post_fork()` configuration fix this?

Reply:
AFAIK, this is not enough. The entire trace pipeline (particularly batch processor) needs to be setup in post_fork for it to work.